experimental/samples/runtime/dotnet/echo-bot/echo-bot/Dialogs/echo/generated/echo.lu.qna.dialog

```diff
@@ -0,0 +1,4 @@
+{
+    "$kind": "Microsoft.CrossTrainedRecognizerSet",
+    "recognizers": []
+}
```

Comment:
Do we expect these samples to be used primarily in Composer? If so, I would remove the generated folder and its contents from both and let it handle producing these directly.

Reply:
Another great question.  One avenue could be you could run the sample directly.  The other is that you use it in Composer.  So... Not sure at this point.